test/index.test.js

```diff
@@ -381,6 +381,95 @@ describe('index', function () {
             });
         });
 
+        it('creates containers from a private docker registry and starts them', () => {
+            container = 'docker-registry.foo.bar:1111/someImage:latest';
+
+            const launcherContainer = {
```

Comment:
This whole const block looks identical to https://github.com/screwdriver-cd/executor-docker/blob/master/test/index.test.js#L112-L173

Any reason we don't move that part out and use them here instead of redefining the exact same const?